### DISCUSSION TITLE
Pull requests for translation downloads don't get created until push

### DISCUSSION THREAD

**Discussion by RandomEngy** [crowdin/github-action#237] [Q&A] [answered]
It appears that the recommended upload/download action only gets run when something is pushed to the branch. That means if you do some work on Crowdin and a bunch of new translations are ready, they don't create a pull request until you make some other push to the branch to kick it off. [My workflow, for reference](https://github.com/RandomEngy/VidCoder/blob/beta/.github/workflows/crowdin.yml).

Is there a way to manually poke the action into creating a pull request without needing to push to the branch?

**Comment by andrii-bodnar** [comment, marked as answer]
There are a number of ways to trigger the translation synchronization (push to branch, cron, manually, webhook, etc). 

See the [Triggers](https://github.com/crowdin/github-action/blob/master/EXAMPLES.md#triggers) examples for more details.

**Reply by RandomEngy** [reply]
Thanks. The `workflow_dispatch:` helped do it manually. Though the docs don't talk about the webhook approach. Would that one let you run the action when the translations were updated?

**Reply by andrii-bodnar** [reply]
There is a guide here - https://store.crowdin.com/github-actions-webhook
